code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -446,7 +461,7 @@
 	return ..()
 
 /datum/dynamic_ruleset/midround/from_ghosts/nuclear/ready(forced = FALSE)
-	if (required_candidates > (dead_players.len + list_observers.len))
+	if (!check_candidates())
```

Comment:
for the record the same could be said of the other three things here that were converted, at least I think

Reply:
I'm going to die

Reply:
I'm so sorry

Reply:
Wait no you're wrong I overrode check_candidates here